sympy/stats/rv.py

```diff
@@ -1300,11 +1324,11 @@ def return_generator_finite():
         while faulty:
             d = ps.sample(size=(numsamples,) + ((size,) if isinstance(size, int) else size),
                           library=library, seed=seed) # a dictionary that maps RVs to values
+
```

Comment:
Remove unnecessary new lines. Please make a separate PR for these kind of changes.